README.md

```diff
@@ -2,20 +2,30 @@
 
 ## Cromwell run
 
-Import summary stats using [import.wdl](wdl/import.wdl). Prepare a list of summary stats like in reference configuration file (one summary stat bucket path per line) [import.json](wdl/import.json).
+Import summary stats using [import.wdl](wdl/import.gwas_pipeline.wdl). Prepare a list of summary stats like in reference configuration file (one summary stat bucket path per line) [import.json](wdl/import.gwas_pipeline.json).
 
 After successful import run, copy generated file to a single bucket using proper file structure using [copy_cromwell_import_to_bucket_puddle.py ](scripts/copy_cromwell_import_to_bucket_puddle.py).
 
-Parameters needed are cromwell hash and path to destination bucket.
-copy_cromwell_import_to_bucket_puddle.py cromwell_hash gs://bucket_for_deployment_pickerupper/v8/
+Parameters needed are cromwell hash and path to destination bucket: `copy_cromwell_import_to_bucket_puddle.py cromwell_hash gs://bucket_for_deployment_pickerupper/v8/`
 
 ***You need to have a socks5 proxy open in localhost:5000 to cromwell machine to get the metadata.***
 
 Example proxy creation if cromwell runs in google VM: `gcloud compute ssh cromwell-machine-name -- -D localhost:5000 -N`.
 
 Alternatively if direct access available change url with `--cromwell_url yourURL` and remove proxy (--socks_proxy "")
 
-## Copy
+## Cromwell run (meta-analysis results)
+
+Use [import.ukbb.wdl](wdl/import.ukbb.wdl).
+
+Use [import.ukbb.json](wdl/import.ukbb.json) as a config template. Most of the variables/files don't need to be changed, but two input files need to be generated:
+
+1. `pheweb_import.summaryfiles`: list of meta-analysis result summary stats (one summary stat bucket path per line)
```

Comment:
can you add example of an input file to the create_custom_json.py

Reply:
and mention that this is to import ukbb,est,fg metas

Reply:
Added example on input and usage of the create_custom_json.py. Also made the script a bit less hardcoded so it can be used in the standard import case as well.